package.json

```diff
@@ -1,8 +1,16 @@
 {
   "name": "athapt",
   "license": "MIT",
-  "private": true,
-  "workspaces": [
-    "packages/*"
-  ]
+
+  "scripts": {
+    "lint": "eslint src/**/*.js"
+  },
+
+  "devDependencies": {
+    "eslint": "^7.10.0",
+    "eslint-config-google": "^0.14.0"
+  },
+  "dependencies": {
+    "@babel/eslint-plugin": "^7.11.5"
```

Comment:
What is this plugin for? because I don't see it used in extends. Or is that for `eslint:recommended`?